deps/rabbit/src/rabbit_db_queue.erl

```diff
@@ -965,6 +966,40 @@ set_in_khepri(Q) ->
     Path = khepri_queue_path(amqqueue:get_name(Q)),
     rabbit_khepri:put(Path, Q).
 
+%% -------------------------------------------------------------------
+%% list_transient().
+%% -------------------------------------------------------------------
+
+-spec list_transient() -> {ok, Queues} | {error, any()} when
+      Queues :: [amqqueue:amqqueue()].
+%% @doc Applies `UpdateFun' to all transient queue records.
+%%
+%% @private
+
+list_transient() ->
+    rabbit_khepri:handle_fallback(
+      #{mnesia => fun() -> list_transient_in_mnesia() end,
+        khepri => fun() -> list_transient_in_khepri() end
+       }).
+
+list_transient_in_mnesia() ->
```

Comment:
sorry for the late comment, but to me it looks like if mnesia is used, this will list all queues, not just transient ones.
I get a "Deprecated features are being used" "transient_nonexcl_queues | In use Permitted by default" warning when I have a single durable quorum queue.

```
> rabbit_db_queue:list_transient().
{ok,[#amqqueue{name = #resource{virtual_host = <<"/">>,
                                kind = queue,name = <<"qq1">>},
               durable = true,auto_delete = false,exclusive_owner = none,
               ...
```

Reply:
Then there is a long-standing bug in this function. Could you please file an issue for it? I can’t work on this one right now.